app/tests/unit/components/Form/ProjectContactForm.test.tsx

```diff
@@ -463,18 +463,31 @@ describe("The ProjectContactForm", () => {
 
     expect(
       componentTestingHelper.environment.mock.getAllOperations()
-    ).toHaveLength(2);
+    ).toHaveLength(3);
 
-    const mutationUnderTest =
+    const firstMutationUnderTest =
       componentTestingHelper.environment.mock.getAllOperations()[1];
 
-    expect(mutationUnderTest.fragment.node.name).toBe(
-      "undoFormChangesMutation"
+    expect(firstMutationUnderTest.fragment.node.name).toBe(
+      "deleteFormChangeWithConnectionMutation"
     );
 
-    expect(mutationUnderTest.request.variables).toMatchObject({
+    expect(firstMutationUnderTest.request.variables).toMatchObject({
+      input: {
+        id: "Form ID 1",
+      },
+    });
+
+    const secondMutationUnderTest =
+      componentTestingHelper.environment.mock.getAllOperations()[2];
+
+    expect(secondMutationUnderTest.fragment.node.name).toBe(
+      "deleteFormChangeWithConnectionMutation"
+    );
+
+    expect(secondMutationUnderTest.request.variables).toMatchObject({
       input: {
-        formChangesIds: [1, 2],
+        id: "Form ID 2",
       },
     });
```

Comment:
we should refactor this using the testing helper, something like that (you will probably need the connection in there, or use an `expect.objectContaining({...})` )
```ts
componentTestingHelper.expectMutationToBeCalled(
  "deleteFormChangeWithConnectionMutation",
  {
    variables: {
      input: {
        id: "Form ID 1"
      }
    }
  });
```